### DISCUSSION TITLE
Option for hiding collections

### DISCUSSION THREAD

**Discussion by warlon** [usebruno/bruno#3031] [Ideas]
I have to often showcase things to customers using Bruno, but I don't want to let them see the the other collections I might have, since they could reveal our other customer names. Adding an option to temporarily hide individual collections would not only improve privacy in this situation, but also could help when you just want to focus on one or few collections.

**Comment by berlingoqc** [comment]
Yes indeed. You can always close them and open them back when needed but it's annoying having to go back to the file picker.

What I propose would be to keep a history of recently open collections so you can close them and open them quickly when needed.

Also it's not good to have to many collections open because it start file watcher for each

**Comment by berlingoqc** [comment]
The current` preference.json` store only the currently opened collection in lastOpenedCollections , when we close one collection should move to another field with list of previously opened collection
```
{
        "window-bounds": {
                "x": 0,
                "y": 44,
                "width": 1668,
                "height": 1048
        },
        "lastOpenedCollections": [
                "/Users/DjoeBlow/Downloads/ZPA",
        ],
        "lastClosedCollections": [ // possible new field 
                "/Users/DjoeBlow/Downloads/i_was_just_closed"
        ]
        "maximized": false
}%         
```
